tools/gm-to-obj/README.md

```diff
@@ -0,0 +1,16 @@
+## .GM to .OBJ file converter
+
+### Requirements
+In order to run this script you need:
+- Python 3+
+- Some .gm files from one game
+
+### How to
+To run the script, just run
+``python .\gm-to-obj.py`` 
```

Comment:
I think the command line interface should be `gm-to-obj.py model_path.gm [-o new_model_path.obj]`

if no output file is specified, then an obj file should be created with the same name (but different file extension) and in the same directory as an input file